### ISSUE TITLE
[🐛 Bug]: ChromiumOptions: ClassCastException when "args" not provided

### ISSUE THREAD

**Issue by kool79** [SeleniumHQ/selenium#12064]
### What happened?

https://github.com/SeleniumHQ/selenium/blob/43eb1e5477c9ef30bf6e2097ef984dec3f25077f/java/src/org/openqa/selenium/chromium/ChromiumOptions.java#LL369C9-L369C97

```
List<String> arguments = (List<String>) (options.getOrDefault("args", new HashMap<>()));
should be fixed to 
List<String> arguments = (List<String>) (options.getOrDefault("args", new ArrayList<>()));
```

you try to cast HashMap to List.

### How can we reproduce the issue?

```shell
.
```

### Relevant log output

```shell
java.lang.ClassCastException: class java.util.HashMap cannot be cast to class java.util.List (java.util.HashMap and java.util.List are in module java.base of loader 'bootstrap')
	at org.openqa.selenium.chromium.ChromiumOptions.mergeInOptionsFromCaps(ChromiumOptions.java:346)
	at org.openqa.selenium.chrome.ChromeOptions.merge(ChromeOptions.java:78)
```

### Operating System

windows

### Selenium version

4.8.0, 4.9.0

### What are the browser(s) and version(s) where you see this issue?

chrome

### What are the browser driver(s) and version(s) where you see this issue?

.

### Are you using Selenium Grid?

.

**Comment by titusfortner** [comment]
That sounds right. Want to make a PR for us? 😄 Thanks.

**Comment by diemol** [comment]
Closed via #12065
